pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -17,10 +18,27 @@ const (
 )
 
 type BitWardenContext struct {
-	BWItem     string        `json:"bw_item"`
-	Field      string        `json:"field,omitempty"`
-	Attachment string        `json:"attachment,omitempty"`
-	Attribute  AttributeType `json:"attribute,omitempty"`
+	BWItem               string                 `json:"bw_item"`
+	Field                string                 `json:"field,omitempty"`
+	Attachment           string                 `json:"attachment,omitempty"`
+	Attribute            AttributeType          `json:"attribute,omitempty"`
+	DockerConfigJSONData []DockerConfigJSONData `json:"dockerconfigJSON,omitempty"`
+}
+
+type DockerConfigJSONData struct {
+	BWItem      string `json:"bw_item"`
+	RegistryURL string `json:"registry_url"`
```

Comment:
Please clarify in these that those refere to a bitwarden field, e.G. by naming it `RegistryURLBItwardenFIeld`, `AuthBiwardenField`, `EmailBitwardenField`